README.md

```diff
@@ -43,7 +43,7 @@ $ python
 ```
 
 ```python
->>> from qiskit import *
+>>> from qiskit.terra import *
```

Comment:
should the readme  just use qiskit import *

Reply:
I need to check if this is possible. In theory, Terra's `qiskit` package should not contain a `__init__.py` or it will conflict with the one provided by the Qiskit metapackage which needs to get some things from `terra`, and some others from `aer`.

If installing the standalone `terra`, I think everything should go under `qiskit.terra`.